dags/hivemind_etl_helpers/ingestion_pipeline.py

```
@@ -68,3 +89,78 @@
+                )
+
+                latest_date = parse(latest_document[0][0].payload[field_name])
+            else:
+                raise ValueError(
+                    f"Index not created successfully! index creation result: {result}"
+                )
+        except Exception as exp:
+            logging.error(f"Error: {exp} while loading latest point!")
+            latest_date = None
+        finally:
+            return latest_date
```

Ruff

`return` inside `finally` blocks cause exceptions to be silenced

(B012)